radio/src/gui/480x272/lcd.cpp

```diff
@@ -280,7 +280,7 @@ void drawTimerMode(coord_t x, coord_t y, int32_t mode, LcdFlags att)
 {
   if (mode >= 0) {
     if (mode < TMRMODE_COUNT)
-      return lcdDrawTextAtIndex(x, y, STR_VTMRMODES, mode, att);
+      lcdDrawTextAtIndex(x, y, STR_VTMRMODES, mode, att);
```

Comment:
I have seen @bsongis add the brackets even for single-line blocks multiple times (even when there's no comment) so I assume it's "always" now :)